doc/source/User_guide/pyaedt_extensions_doc/hfss/push_excitation.rst

```diff
@@ -0,0 +1,55 @@
+Push excitation from file
+=========================
+
+The **push excitation from file** extension allows users to assign a time-domain excitation to a port in an HFSS design
+by importing data from a file.
+
+The extension provides a graphical user interface (GUI) for configuration,
+or it can be used in batch mode via command line arguments.
+
+Features
+--------
+
+- Automatically detects and lists available ports in the active HFSS design.
+- Allows users to browse and select a time-domain excitation file.
+- Supports both light and dark themes for the GUI.
+- Assigns excitations programmatically using a batch-mode interface.
+- Validates file paths and port selections to ensure proper configuration.
```

Comment:
@Samuelopez-ansys for me here (and in all the other files) shouldn't be in third person.

Reply:
Why? They are the extension features.